src/analysis/completion.ml

```diff
@@ -585,11 +589,14 @@ let complete_prefix ?get_doc ?target_type ?(kinds = []) ~keywords ~prefix
     in
     let base_completion =
       match (is_label : is_label) with
-      | `No -> []
-      | `Maybe -> Env.fold_labels add_label_description prefix_path env []
-      | `Description lbls ->
+      | No -> []
+      | Maybe ->
+        Env.fold_labels Legacy add_label_description prefix_path env []
```

Comment:
Does it make sense to make a function the folds over both flavors of label? That will make it less likely to skip one flavor.

Reply:
done